src/v2/services/implementations/DappStakingService.ts

```diff
@@ -49,12 +49,39 @@ export class DappStakingService implements IDappStakingService {
     return new TvlModel(tvl, tvlDefaultUnit, tvlUsd);
   }
 
+  public async nominationTransfer({
```

Comment:
Maybe to write test for this method. Although, there is no much logic to test here you can check if repo method is called with proper parameters, also you can test the case when not parameters are provided

Reply:
Sure, how about like [this?](https://github.com/AstarNetwork/astar-apps/pull/533/commits/1cf3057c67ef6bf819f4f6755166c0bfaaec9cea)